compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skiko.kt

```diff
@@ -446,6 +451,31 @@ class SkikoComposeUiTest @InternalTestApi constructor(
         override fun updateState(oldValue: TextFieldValue?, newValue: TextFieldValue) = Unit
     }
 
+    private inner class TestDragAndDropManager : PlatformDragAndDropManager {
+        override val isRequestDragAndDropTransferRequired: Boolean
+            get() = true
+
+        override fun requestDragAndDropTransfer(
+            source: PlatformDragAndDropSource,
+            offset: Offset
+        ) {
+            val startTransferScope = object : PlatformDragAndDropSource.StartTransferScope {
+                override fun startDragAndDropTransfer(
+                    transferData: DragAndDropTransferData,
+                    decorationSize: Size,
+                    drawDragDecoration: DrawScope.() -> Unit
+                ): Boolean {
+                    return true
+                }
+            }
+            with(source) {
+                startTransferScope.startDragAndDropTransfer(offset) {
+                    true
```

Comment:
It's incorrect. It should be true only after the first call of `scope.startDragAndDropTransfer`

Reply:
Fixed.